### DISCUSSION TITLE
Can we choose which saved settings to apply to the secure screen?

### DISCUSSION THREAD

**Discussion by cary-rowen** [nvaccess/nvda#12553] [Ideas]
Hello,

As the title says, even if not all configurations are selectable, at least we should be able to choose which add-ons to apply to the secure screen, because some add-ons have no meaning on the secure screen, and these add-ons will Occupies extra hard disk space and reduces NVDA performance.
I know that you can delete unnecessary add-ons in the "systemConfig" directory under the NVDA installation directory, but this is not friendly. For those newbies, they may not know this.

Thanks.

**Comment by ABuffEr** [comment]
Hi,
I agree. It could be introduced a button to switch to systemConfig profile, under General options. This could let to modify settings and choose add-ons without big changes in NVDA, I think.

**Comment by dpy013** [comment]
It might be nice if NVDA could select the items to be backed up when exporting the configuration.
For a specific example, you can refer to reaper export configuration function.

**Comment by vgjh2005** [comment]
Hi:
I think we could consider to create a called "configuration filter dialog" to select what configuration should be sync after clicking "Use currently saved settings during sign-in and on secure screens (requires administrator privileges)" button.

**Comment by josephsl** [comment]
Hi,

This was discussed before, and @NVDAEs was trying to work on this issue a while back.

**Reply by nvdaes** [reply]
Hello, I worked in a PR mentioned in issue #6305
Hope this can be implemented.

**Comment by cary-rowen** [comment]
@josephsl @nvdaes 
Hello,
This is great, hopefully this feature will be implemented soon, I was shocked to see that #6305 was proposed a few years ago.
